github/lib/repoDataMessage.ts

```diff
@@ -64,6 +64,12 @@ export async function repoDataMessage({
         });
     }
 
+    let description: string | null = resData.description;
+
+    if(description === null){
+        description = 'No description provided.';
+    }
+
```

Comment:
Hey @Nabhag8848 , thanks for solving this for us. Works well  :rocket:works !

But maybe we can do something like this 
` const description = resData.description || 'No description provided.';` 

This will make sure that the description is const and is not updated before sending it to the main channel. Also it fits with the code style we have in this file and saves some lines.

Reply:
@Nabhag8848  Can this PR be extended to fix some other places where we are showing null values ?

Reply:
Yep! we can do that. Do you encounter any nulls, possibly i didn't encounter yet? if you have point me up.

Reply:
Hey, @samad-yar-khan . I found lot of places where there is asyncronous operation without `await` and vice versa. should i make the commits within this PR? also we can merge my PR #41 branch with this one to make it bigger or we will include issue #47  within PR #41 later. 
How should we proceed.